src/client/components/pages/parts/revisions-table.js

```diff
@@ -42,41 +42,62 @@ function RevisionsTable(props) {
 				className={tableCssClasses}
 			>
 				<thead>
-					<tr>
-						<th className="col-sm-2">Revision ID</th>
-						<th className="col-sm-5">Modified entities</th>
-						<th className="col-sm-3">User</th>
-						<th className="col-sm-2">Date</th>
-
-					</tr>
+					{
+						results.length > 0 ?
+							<tr>
+								<th className="col-sm-2">Revision ID</th>
```

Comment:
I wonder about these column sizes.
I've tried removing all the col-… classes on the th elements, and there are positive and negative aspects:

Positive: Better use of space, for example rev id is as small as it can while giving space to the large notes

Negative: Depending on content the columns are not the same size, which doesn't look great when going back or forth a page.
Notes often contain urls, which have no spaces, and as such will take as much space as possible, making the 'modified entities' column wrap first. Not sure I like that.

I'm conflicted about this. Let's keep it as-is for now, but if you're interested, keep that issue in the back of your mind.
Maybe there is a solution out there that allows us to set min- or max-width on certain columns (useful for example for rev id column [small] and notes column [we don't want it to be wider than, say, 40%])